release-calendar/src/client/components/Square.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+
+export interface SquareProps {
+  channel: string;
+}
+export class Square extends React.Component<SquareProps, {}> {
```

Comment:
> the new Square.tsx which is really only separated from the channelTable so that it's styling can also be separated

It's not necessary for this to be a separate component just for styling. CSS supports this just fine.

Reply:
Before, I had been using a ts function that adds styling when a row is selected. It was pretty patchy so I got rid of the regular buttons and replaced it with checkboxes. Now when a row is selected, the row will change appearance only though css.